### ISSUE TITLE
AppSyncApi:  normalizeResolverKey case conversion breaks deploy

### ISSUE THREAD

**Issue by w5mix** [serverless-stack/serverless-stack#505]
When defining a type such as..
```
type MyType {
   name: String
}
```
And a resolver, such as..
```
"MyType name": "src/name.main"
```

`normalizeResolverKey` converts the key from `MyType` to `Mytype` (making only the first character uppercase), which results in this error upon deploy... 

```
failed: Type Mytype not found (Service: AWSAppSync; Status Code: 404; Error Code: NotFoundException; Request ID: 9a9c9a91-f514-4252-a3e7-aa3c8cbd0011; Proxy: null)
```

Link to `normalizeResolverKey` below...
https://github.com/serverless-stack/serverless-stack/blob/67a334a9686a50f8ee276852302a9220f6f14815/packages/resources/src/AppSyncApi.ts#L379

**Comment by fwang** [comment]
Hi @w5mix, thanks for bringing this up. I will try to put in a fix this week.

**Comment by w5mix** [comment]
@fwang Thank you, Frank!

**Comment by fwang** [comment]
No worries! Just released [0.31.0](https://github.com/serverless-stack/serverless-stack/releases/tag/v0.31.0) with the fix
